tests/console/journalctl.pm

```diff
@@ -117,14 +117,22 @@ sub run {
     assert_script_run('journalctl --vacuum-size=100M');
     assert_script_run('journalctl --vacuum-time=1years');
     # Rotate once more and verify the journal afterwards
-    record_soft_failure "bsc#1171858" if (script_run('journalctl --verify --verify-key=`cat /root/journalctl-setup-keys.txt`') != 0);
+    record_soft_failure "bsc#1171858" if (script_run('journalctl --verify --verify-key=`cat /var/tmp/journalctl-setup-keys.txt`') != 0);
     assert_script_run('journalctl --rotate');
-    record_soft_failure "bsc#1171858" if (script_run('journalctl --verify --verify-key=`cat /root/journalctl-setup-keys.txt`') != 0);
+    record_soft_failure "bsc#1171858" if (script_run('journalctl --verify --verify-key=`cat /var/tmp/journalctl-setup-keys.txt`') != 0);
 }
 
 sub post_fail_hook {
     script_run('journalctl -x > /var/tmp/journalctl.txt');
     upload_logs('/var/tmp/journalctl.txt');
+    script_run('rm -f /var/tmp/journalctl.txt');
+}
+
+sub post_run_hook {
+    script_run('rm -f /var/tmp/journalctl.txt');
+    script_run('rm -f /var/tmp/journalctl-setup-keys.txt');
+    script_run('rm -f /var/tmp/journal_serial.out');
+    script_run('rm -f /var/tmp/reboottime');
```

Comment:
Why only delete them here? Shouldn't they also be deleted in `post_fail_hook`

Reply:
Updated accordingly